server/utils/routes.js

```diff
@@ -10,4 +11,14 @@ export const routes = {
   resetPwNoUser: `${basePath}/resetpw?e=nouser`,
   resetPwError: `${basePath}/resetpw?e=db`,
   resetPwErrorUnmatched: `${basePath}/resetpw?e=unmatched`,
+  login: `${basePath}/login`,
+  loginWithError: (err) => `${basePath}/login?e=${err}`,
+  loginNAError: `${basePath}/login?e=NA`,
```

Comment:
can this use `loginWithError` and pass "NA"?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/241/commits/be3741398c54927adbe373fa7b03098b9ba0424c